--- a/alsa-lib-1.2.4/modules/mixer/simple/python.c
+++ b/alsa-lib-1.2.4/modules/mixer/simple/python.c
@@ -22,6 +22,7 @@
 #include "Python.h"
 #include <stddef.h>
 #include <limits.h>
+#include <linux/limits.h>
 #include "config.h"
 #include "asoundlib.h"
 #include "mixer_abst.h"

--- a/alsa-lib-1.2.4/src/conf.c
+++ b/alsa-lib-1.2.4/src/conf.c
@@ -418,6 +418,7 @@ beginning:</P>
 #include <stdarg.h>
 #include <stdbool.h>
 #include <limits.h>
+#include <linux/limits.h>
 #include <sys/stat.h>
 #include <dirent.h>
 #include <locale.h>

--- a/alsa-lib-1.2.4/src/dlmisc.c
+++ b/alsa-lib-1.2.4/src/dlmisc.c
@@ -33,6 +33,7 @@
 #include <pthread.h>
 #endif
 #include <limits.h>
+#include <linux/limits.h>
 
 #if defined(HAVE_LIBDL) && defined(__GLIBC__) && !defined(__UCLIBC__)
 #define DL_ORIGIN_AVAILABLE 1

--- a/alsa-lib-1.2.4/src/topology/data.c
+++ b/alsa-lib-1.2.4/src/topology/data.c
@@ -20,6 +20,7 @@
 #include "list.h"
 #include "tplg_local.h"
 #include <ctype.h>
+#include <linux/limits.h>
 
 #define UUID_FORMAT "\
 %02x:%02x:%02x:%02x:%02x:%02x:%02x:%02x:\

--- a/alsa-lib-1.2.4/src/ucm/main.c
+++ b/alsa-lib-1.2.4/src/ucm/main.c
@@ -36,6 +36,7 @@
 #include <pthread.h>
 #include <sys/stat.h>
 #include <limits.h>
+#include <linux/limits.h>
 
 /*
  * misc

--- a/alsa-lib-1.2.4/src/ucm/parser.c
+++ b/alsa-lib-1.2.4/src/ucm/parser.c
@@ -33,6 +33,7 @@
 #include "ucm_local.h"
 #include <dirent.h>
 #include <limits.h>
+#include <linux/limits.h>
 
 /* Directories to store UCM configuration files for components, like
  * off-soc codecs or embedded DSPs. Components can define their own

--- a/alsa-lib-1.2.4/src/ucm/ucm_subs.c
+++ b/alsa-lib-1.2.4/src/ucm/ucm_subs.c
@@ -28,6 +28,7 @@
 #include <stdbool.h>
 #include <sys/stat.h>
 #include <limits.h>
+#include <linux/limits.h>
 
 static char *rval_open_name(snd_use_case_mgr_t *uc_mgr)
 {
