--- a/glibc-2.13/include/features.h
+++ b/glibc-2.13/include/features.h
@@ -320,11 +320,17 @@
 #endif
 
 /* We do support the IEC 559 math functionality, real and complex.  */
-#define __STDC_IEC_559__		1
-#define __STDC_IEC_559_COMPLEX__	1
+#ifndef __STDC_IEC_559__
+#  define __STDC_IEC_559__		1
+#endif
+#ifndef __STDC_IEC_559_COMPLEX__
+#  define __STDC_IEC_559_COMPLEX__	1
+#endif
 
 /* wchar_t uses ISO 10646-1 (2nd ed., published 2000-09-15) / Unicode 3.1.  */
-#define __STDC_ISO_10646__		200009L
+#ifndef __STDC_ISO_10646__
+#  define __STDC_ISO_10646__		200009L
+#endif
 
 /* This macro indicates that the installed library is the GNU C Library.
    For historic reasons the value now is 6 and this will stay from now
