diff --git a/xtrans-1.2.6/Makefile.in b/xtrans-1.2.6/Makefile.in
index 161ddd71..e905eb2e 100644
--- a/xtrans-1.2.6/Makefile.in
+++ b/xtrans-1.2.6/Makefile.in
@@ -269,7 +269,7 @@ Xtransinclude_HEADERS = \
 
 aclocaldir = $(datadir)/aclocal
 aclocal_DATA = xtrans.m4
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = xtrans.pc
 MAINTAINERCLEANFILES = ChangeLog INSTALL
 EXTRA_DIST = ${aclocal_DATA}
